internal/wasm/func_validation.go

```diff
@@ -1300,6 +1300,23 @@ func (m *Module) validateFunctionWithMaxStackValues(
 					return fmt.Errorf("cannot pop the operand for %s: %v", vectorInstructionName[vecOpcode], err)
 				}
 				valueTypeStack.push(ValueTypeV128)
+			case OpcodeVecI8x16Eq, OpcodeVecI8x16Ne, OpcodeVecI8x16LtS, OpcodeVecI8x16LtU, OpcodeVecI8x16GtS,
```

Comment:
too bad we don't have a mask or range here.. is this contiguous?

Reply:
not contiguous at all unfortunately :D